src/components/VersionSelect.tsx

```diff
@@ -162,7 +162,9 @@ export const VersionSelect = ({
         sx: {
           padding: "7px 9px !important",
           backgroundColor: palette.common.white,
-          borderRadius: "0px"
+          borderRadius: "0px",
+          fontFamily: "monospace",
+          fontSize: "13px"
```

Comment:
This controls the font style for the version displayed in the box that opens the dropdown